vms/platformvm/genesis/camino.go

```diff
@@ -88,3 +90,13 @@ func (b *Block) Txs() []*txs.Tx {
 	txs = append(txs, b.Deposits...)
 	return append(txs, b.UnlockedUTXOsTxs...)
 }
+
+// Generate deposit offer id from its bytes hash and set it to offer's ID field
+func SetDepositOfferID(offer *deposit.Offer) error {
+	bytes, err := blocks.GenesisCodec.Marshal(blocks.Version, offer)
```

Comment:
I suggest txs.Version, it s more tx related than blocks

Reply:
To late for this, its func for id for genesis offers only. We did it with genesis blocks codec, this PR just moves this func from deposit package to genesis, cause its no longer part of offers logic, new offers ids are add deposit tx ids